wu-master-management.csl

```diff
@@ -0,0 +1,652 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" name-delimiter="/" et-al-min="4" et-al-use-first="3" et-al-subsequent-min="4" et-al-subsequent-use-first="3" initialize="false" initialize-with="." demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>WU Master Management</title>
+    <id>http://www.zotero.org/styles/wu-master-management.csl</id>
```

Comment:
Style IDs shouldn't have ".csl" extensions (e.g. `<id>http://www.zotero.org/styles/wu-master-management</id>` instead of `<id>http://www.zotero.org/styles/wu-master-management.csl</id>`)